### DISCUSSION TITLE
Ring Intercom - missing ring notification

### DISCUSSION THREAD

**Discussion by lastfreeusernameicouldfind** [tsightler/ring-mqtt#858] [Support Q&A]
Hi all,

I do have my ring device setup and (partially) working with ring-mqtt. I do get the events LOCKED, UNLOCKED. I am also able to unlock via mqtt.
Unfortunately I do not get ring notifications (means when someone rings at the door). Which mqtt topic should be triggered for ring events? Any idea what could be missing or wrong?

Thanks and best regards
Dom

**Comment by tsightler** [comment]
There should be a ding/state topic with any ding events, same as a camera.  If you are not receiving push notifications you can stop the addon, delete the account from Ring Control Center and re-authenticate, that usually gets it going again.

**Comment by lastfreeusernameicouldfind** [comment]
In the original ring app I am getting push notifications when someone rings at the door. Problem is only via ring-mqtt. 
Need to try if I am getting a ding/state in addition to the documented ones (The MQTT-Devices Topics only describe a lock without ding/state: https://github.com/tsightler/ring-mqtt/wiki/MQTT-Device-Topics#locks). Will check and let you know. Thanks

**Reply by lastfreeusernameicouldfind** [reply]
I can confirm ding/state works with ring intercom.
Thanks!
